.github/workflows/test-eclhelper.yml

```diff
@@ -0,0 +1,19 @@
+name: Check that eclhelper interface has not changed
+
+on:
+  pull_request:
+    branches:
+      - "master"
+      - "candidate-*"
+
+jobs:
+  build:
+    name: "Check eclhelper interface"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
```

Comment:
will this mean it will not spot a change, if a multi-commit PR is pushed, unless the eclhelper.hpp changed is in the final commit?

Reply:
No (I wondered that too). HEAD is the merge commit, and HEAD^ is the original base branch, regardless of how many commits there are in the PR branch.